`make install` has failed

# What did you do? (required. The issue will be **closed** when not provided.)

Deployed the vuls manually according to the docs, and `make install` from the master branch.
https://vuls.io/docs/ja/install-manually.html#deploy-vuls

# What did you expect to happen?

The software is installed successfully.

# What happened instead?

Install has failed.

This is probably due to #1080 . `./cmd/vuls/main.go` does not exist in the repository. The commit ID at this time is  https://github.com/future-architect/vuls/tree/5fea4eaef823b7ec9c99a165b5f532540e25c998 .


* Current Output

```
make: *** No rule to make target `cmd/vuls/main.go', needed by `install'.  Stop.
```

# Steps to reproduce the behaviour

```
$ git clone https://github.com/future-architect/vuls.git
$ cd vuls
$ make install
```

# Configuration (**MUST** fill this out):

* Go version (`go version`):

```
go version go1.15.5 darwin/amd64
```

* Go environment (`go env`):

Some values are private, so we have rewritten them to dummy values.

```
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/chroju/Library/Caches/go-build"
GOENV="/Users/chroju/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOINSECURE=""
GOMODCACHE="/Users/chroju/repos/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="darwin"
GOPATH="/Users/chroju/repos"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/Users/chroju/.asdf/installs/golang/1.15.5/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/Users/chroju/.asdf/installs/golang/1.15.5/go/pkg/tool/darwin_amd64"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
GOMOD="/Users/chroju/repos/src/github.com/future-architect/vuls/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/z5/c1_273xs4vbcyjjwh1t52_0xwnh588/T/go-build631130376=/tmp/go-build -gno-record-gcc-switches -fno-common"
```

* Vuls environment:

N/A

* config.toml:

N/A

* command:

N/A

--- a/.gitignore
+++ b/.gitignore
@@ -1,4 +1,3 @@
-vuls
 .vscode
 *.txt
 *.json

--- a/cmd/vuls/main.go
+++ b/cmd/vuls/main.go
@@ -1,0 +1,38 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"context"
+
+	"github.com/future-architect/vuls/config"
+	commands "github.com/future-architect/vuls/subcmds"
+	"github.com/google/subcommands"
+)
+
+func main() {
+	subcommands.Register(subcommands.HelpCommand(), "")
+	subcommands.Register(subcommands.FlagsCommand(), "")
+	subcommands.Register(subcommands.CommandsCommand(), "")
+	subcommands.Register(&commands.DiscoverCmd{}, "discover")
+	subcommands.Register(&commands.TuiCmd{}, "tui")
+	subcommands.Register(&commands.ScanCmd{}, "scan")
+	subcommands.Register(&commands.HistoryCmd{}, "history")
+	subcommands.Register(&commands.ReportCmd{}, "report")
+	subcommands.Register(&commands.ConfigtestCmd{}, "configtest")
+	subcommands.Register(&commands.ServerCmd{}, "server")
+
+	var v = flag.Bool("v", false, "Show version")
+
+	flag.Parse()
+
+	if *v {
+		fmt.Printf("vuls %s %s\n", config.Version, config.Revision)
+		os.Exit(int(subcommands.ExitSuccess))
+	}
+
+	ctx := context.Background()
+	os.Exit(int(subcommands.Execute(ctx)))
+}

